bin/helpers/atsHelper.js

```diff
@@ -108,7 +108,9 @@ exports.patchCypressConfigFileContent = (bsConfig) => {
 
     let confPath = bsConfig.run_settings.cypress_config_file;
     let patchedConfPathList = confPath.split(path.sep);
-    patchedConfPathList[patchedConfPathList.length - 1] = 'patched_ats_config_file.js'
+    const uniqueNamePatchFileName = `patched_ats_config_file_${uuidv4()}.js`;
+    patchedConfPathList[patchedConfPathList.length - 1] = uniqueNamePatchFileName;
+    logger.info("Patch file name is " + uniqueNamePatchFileName);
```

Comment:
Why adding this log line, it will be visible to user? Let's change it to debug if it's required for debugging.